src/Kernel/Aggregates/Charge.php

```diff
@@ -369,6 +369,11 @@ public function getTransactions()
         return $this->transactions;
     }
 
+    public function failed()
+    {
+        $this->status = ChargeStatus::failed();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Mundipagg\Core\Kernel\ValueObjects\ChargeStatus' in method 'failed'.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)